kevlar/tests/data/nodiff.contig.fa

```diff
@@ -1,4 +1,3 @@
 >contig1
-ATCTCCCGGAAGACCCTTACAAGCGAGCCAAGGTCCGATTTTGGGTAGATTTCGTCAACAAAAAGGTTACTGCCCCCTCT
-TCCTCTTCCATGTCTGTTTTCTTAACCTAATCCCAGAACACTCTGCCTGTTCAAGCTCATGGAGTCTTAAGTGCAGGTTG
-TGCCAATTATGCATACCATCGTAACCTTGAAAGGCGATGAGCAGAAGACGGCAGCGGACGAGTTCGGAGGGTATCTACAG
+ATCTCCCGGAAGACCCTTACAAGCGAGCCAAGGTCCGATTTTGGGTAGATTTCGTCAACAAAAAGGTTACTGCCCCCTCTTCCTCTTCCATGTCTGTTTTCTTAACCTAATCCCAGAACACTCTGCCTGTTCAAGCTCATGGAGTCTTAAGTGCAGGTTGTGCCAATTATGCATACCATCGTAACCTTGAAAGGCGATGAGCAGAAGACGGCAGCGGACGAGTTCGGAGGGTATCTACAG
+                                                            AAAAGGTTACTGCCCCCTCTTCCTCTTCCAT          5 0 1#
```

Comment:
Add an interesting k-mer annotation so this isn't labeled as a "passenger" call.